examples/lighting-app/efr32/BUILD.gn

```diff
@@ -38,10 +38,6 @@ declare_args() {
   # Dump memory usage at link time.
   chip_print_memory_usage = false
 
-  # PIN code for PASE session establishment.
```

Comment:
Do the EFR32 apps get built with the same PIN and Discriminator by default?

Reply:
Yes by default it will use the test values until the user inject a set in nvm3 with the tool